src/plugins/footnotes/index.ts

```diff
@@ -211,6 +211,16 @@ export default () => {
               })
             )
           }
+          if (node.type === schema.nodes.table_element_footer) {
+            if (node.firstChild?.type === schema.nodes.paragraph) {
```

Comment:
This doesn't sound like a good place to add a paragraph placeholder, as this plugin is for footnotes.

I believe we already have a placeholder plugin that might work for you.

Reply:
done